plextraktsync/trakt/TraktUserList.py

```diff
@@ -51,7 +51,15 @@ def items(self):
 
     @staticmethod
     def build_dict(pl: PublicList):
-        return {(f"{le.type}s", le.trakt): le.rank for le in pl if le.type in ["movie", "episode"]}
+        items = {}
+        for le in pl:
+            if le.type in ["movie", "episode"]:
+                items[(f"{le.type}s", le.trakt)] = le.rank
+            elif le.type == "season" and len(le.item.episodes) > 0:
+                episode_rank = 1 / len(le.item.episodes)
```

Comment:
describe in code comment what this division and later multiplication does

Reply:
Added a comment describing this. Let me know if it's not clear enough or too verbose.

Reply:
I don't understand why right now, but seems good explanation